build.fsx

```diff
@@ -3,15 +3,20 @@
 // --------------------------------------------------------------------------------------
 
 #r @"packages/FAKE/tools/FakeLib.dll"
-
-open Fake
-open Fake.AppVeyor
+open Fake.BuildServer
```

Comment:
I'm not against this, but was it necessary to upgrade to FAKE 5 as part of this PR? Are there any breaking changes?

Reply:
I will split the Pull requests.